sdk/recoveryservicessiterecovery/arm-recoveryservices-siterecovery/samples-dev/operationsListSample.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { SiteRecoveryManagementClient } from "@azure/arm-recoveryservices-siterecovery";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to Operation to return the list of available operations.
+ *
+ * @summary Operation to return the list of available operations.
+ * x-ms-original-file: specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-11-01/examples/Operations_List.json
+ */
+async function returnsTheListOfAvailableOperations() {
+  const resourceGroupName = "resourceGroupPS1";
+  const subscriptionId = "c183865e-6077-46f2-a3b1-deb0f4f4650a";
+  const credential = new DefaultAzureCredential();
+  const client = new SiteRecoveryManagementClient(
+    credential,
+    resourceGroupName,
+    subscriptionId
```

Comment:
is this error from swagger?
https://github.com/Azure/azure-rest-api-specs/blob/5dedb96183ff83552172039606e67eccf0b7ee20/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-11-01/service.json#L40